KomaMRIBase/src/datatypes/Sequence.jl

```diff
@@ -408,15 +408,16 @@ Returns the RF pulses and the delta frequency.
 - `Δf_rf`: (`1-row ::Matrix{Float64}`, `[Hz]`) delta frequency vector
 """
 get_rfs(seq::Sequence, t) = begin
+    ϵ = MIN_RISE_TIME
 	#Amplitude
 	A  = seq.RF.A
 	Δf = seq.RF.Δf
 	#Timings
 	T = seq.RF.T
 	delay = seq.RF.delay
 	T0 = cumsum([0; durs(seq)], dims=1)
-	(sum([⏢(A[1,i], t.-T0[i],sum(T[i]).-2EPS,EPS,EPS,delay[i]) for i=1:length(seq)]),
-	 sum([⏢(Δf[1,i],t.-T0[i],sum(T[i]).-2EPS,EPS,EPS,delay[i]) for i=1:length(seq)])
+	(sum([⏢(A[1,i], t.-T0[i],sum(T[i]).-2ϵ,ϵ,ϵ,delay[i]) for i=1:length(seq)]),
+	 sum([⏢(Δf[1,i],t.-T0[i],sum(T[i]).-2ϵ,ϵ,ϵ,delay[i]) for i=1:length(seq)])
```

Comment:
Not following Julia Blue
```suggestion
	(sum([⏢(A[1,i], t .- T0[i], sum(T[i]) .- 2ϵ, ϵ, ϵ, delay[i]) for i=1:length(seq)]),
	 sum([⏢(Δf[1,i], t .- T0[i], sum(T[i]) .- 2ϵ, ϵ, ϵ, delay[i]) for i=1:length(seq)])
```

Reply:
Addressed in https://github.com/cncastillo/KomaMRI.jl/pull/241/commits/6c2507cf6077ffacc5730b20c24184763332d45f